bindings/kotlin/src/main/kotlin/example.kt

```diff
@@ -0,0 +1,51 @@
+import kotlinx.coroutines.runBlocking
+import uniffi.iota_sdk_ffi.GraphQlClient
```

Comment:
Is the `uniffi` namespace here something we can change?

Reply:
Yes good point